tests/e2e/framework/common.go

```diff
@@ -476,6 +478,10 @@ func (f *Framework) deleteScanSettings(name string) error {
 }
 
 func (f *Framework) createMachineConfigPool(n string) error {
+	if f.Platform == "rosa" {
```

Comment:
I'd like to find a cleaner way to deal with these conditionals in case other platforms come in the future.